ibis/backends/clickhouse/tests/snapshots/test_functions/test_column_regexp_extract/out.sql

```diff
@@ -1,9 +1,9 @@
 SELECT
   CASE
     WHEN notEmpty(
-      extractGroups(CAST("t0"."string_col" AS String), CONCAT('(', '[\d]+', ')'))[3 + 1]
+      extractGroups(CAST("t0"."string_col" AS String), CONCAT('(', '[\\d]+', ')'))[3 + 1]
     )
-    THEN extractGroups(CAST("t0"."string_col" AS String), CONCAT('(', '[\d]+', ')'))[3 + 1]
+    THEN extractGroups(CAST("t0"."string_col" AS String), CONCAT('(', '[\\d]+', ')'))[3 + 1]
```

Comment:
is the `\\` expected?

Reply:
Yep. ClickHouse automatically rewrites single backslashes to double it seems.

Reply:
Sorry, to be clear, I meant his happens in clickhouse itself:

```
ClickHouse local version 24.3.2.1.

localhost :) select extractGroups('1234', '([\d])+');

SELECT extractGroups('1234', '([\\d])+')

Query id: 59b5b1e3-ed46-411e-bb07-4cf9a595643a

   ┌─extractGroups('1234', '([\\d])+')─┐
1. │ ['4']                             │
   └───────────────────────────────────┘

1 row in set. Elapsed: 0.001 sec.

localhost :) select extractGroups('1234', '([\\d])+');

SELECT extractGroups('1234', '([\\d])+')

Query id: e4cb574c-7b9b-4da3-af00-eec2deaf27aa

   ┌─extractGroups('1234', '([\\d])+')─┐
1. │ ['4']                             │
   └───────────────────────────────────┘

1 row in set. Elapsed: 0.001 sec.
```